test/Microsoft.AspNet.Mvc.Razor.Test/LanguageViewLocationExpanderTest.cs

```diff
@@ -14,6 +14,26 @@ public static IEnumerable<object[]> ViewLocationExpanderTestDataWithExpectedValu
             {
                 yield return new object[]
                 {
+                    LanguageViewLocationExpanderOption.Suffix,
+                    new[]
+                    {
+                        "/Views/{1}/{0}.cshtml",
+                        "/Views/Shared/{0}.cshtml"
+                    },
+                    new[]
+                    {
+                        "/Views/{1}/en-GB.{0}.cshtml",
```

Comment:
A suffix keeps it inline with how we had this feature work before, e.g. for mobile views in MVC 4 & 5.